zoloto/coords.py

```diff
@@ -29,14 +39,49 @@ class ThreeDCoordinates(NamedTuple):
 
 class Spherical(NamedTuple):
     """
-    :param float rot_x: Rotation around the X-axis, in radians
-    :param float rot_y: Rotation around the Y-axis, in radians
-    :param float dist: Distance
+    Spherical coordinates, rotated onto their side.
+
+    This is comparable to the ISO convention for spherical coordinates, applied
+    to our rotated axes. Here θ is measured down from the y-axis (rather than
+    the usual z-axis) while φ is measured around the y-axis.
```

Comment:
As it stands, yes this is a function of the labelling of our axes. Whether changing that would be better I don't know. I now suspect that this may have been part of why `libkoki`'s spherical coordinates seemed so unusual, though I'm not completely sure that they were aligned to the normal convention either.